WordPress/src/main/java/org/wordpress/android/ui/posts/editor/ImageEditorInitializer.kt

```diff
@@ -64,5 +70,21 @@ class ImageEditorInitializer {
             } else {
                 throw(IllegalArgumentException(IMAGE_STRING_URL_MSG))
             }
+
+        private fun trackData(): (Pair<TrackableEvent, TrackableAction?>) -> Unit = { data ->
```

Comment:
NitPick: We could simplify this by using kotlin's feature: 
Replace
```
data -> 
           val event = data.first
            val action = data.second
```
with
`(event, action) -> 
`

Wdyt?